L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -284,12 +330,50 @@ class L1TrackObjectNtupleMaker : public edm::one::EDAnalyzer<edm::one::SharedRes
   std::vector<float>* m_gen_z0;
   std::vector<float>* m_gen_mother_pdgid;
 
+  //displaced vertices
```

Comment:
Do all of these need to be added to the ntuples or can you derive most of these in your training code? How much are these variables specific to your use case or are they general for many different algorithm developments?

Reply:
The bare minimum would be the two track indices, the vertex score, and the isReal boolean. The rest could be calculated by remaking the vertex from the two tracks. What I would prefer to do is just remove unused variables (opening angle, parent pt, inTraj, dvEmuFixed branches) and then consolidate the dv and dvEmu branches into one set of branches which is filled with the simulated or emulated values depending on a boolean. That would leave 11 branches which are only saved if a user turns the displaced vertexing on and they would not have to recalculate the vertices. Would that be acceptable?